### PR TITLE
Class02 과제1 Arduino와 Python을 이용한 LED 제어 - JooeunPark

### PR DESCRIPTION
**Pull request by jjooeun**

### CODE REVIEW

**Review by litcoder — APPROVED** [review]
수고 많으셨습니다. :)

File: class02/homework/JooeunPark/hw1/led.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+def main():
+    while(True):
+        n = int(input("input number:"))
+        if n == 1:
+            ctrl.system_start()
+        elif n == 2:
+            ctrl.system_stop()
+        elif n == 3:
+            if ctrl.red is False:
+                ctrl.red = False
```
[the remark this answers is not in the document]
Comment (jjooeun) [reply class02/homework/JooeunPark/hw1/led.py:12]: I think when 'ctrl.red' is called, 'ctrl.red' is toggled. So if 'ctrl.red' is True and I want to change 'ctrl.red' to False, i think i must call 'ctrl.red' twice. Am I wrong..?

[the remark this answers is not in the document]
Comment (jjooeun) [reply class02/homework/JooeunPark/hw1/led.py:12]: I can't write korean because of setting in ubuntu... Excuse me please.. And please reply in korean.

**Comment by litcoder** [comment]
제출 한 답변은 틀리지 않았습니다. 제가 삭제하고 보냈어야 하는데 실수 했네요. :(7/11/23 오후 5:31, Judy ***@***.***> 작성:﻿
@jjooeun commented on this pull request.



In class02/homework/JooeunPark/hw1/led.py:
> @@ -0,0 +1,41 @@
+from iotdemo import FactoryController
+
+def main():
+    while(True):
+        n = int(input("input number:"))
+        if n == 1:
+            ctrl.system_start()
+        elif n == 2:
+            ctrl.system_stop()
+        elif n == 3:
+            if ctrl.red is False:
+                ctrl.red = False

I can't write korean because of setting in ubuntu... Excuse me please.. And please reply in korean.

**Pull request merged**